src/Stryker.Core/Stryker.Core/Initialisation/IProjectAndTest.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace Stryker.Core.Initialisation;
+
+public interface IProjectAndTest
```

Comment:
```suggestion
public interface IProjectAndTests
```

Reply:
I will do a quick PR for that, because it will impact several files, I think
but original naming is not good, agreed.